src/shims/x86/mod.rs

```diff
@@ -146,6 +146,13 @@ pub(super) trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 );
             }
 
+            "pclmulqdq" => {
```

Comment:
Please move this up to the other concrete intrinsic implementations. Down here we only dispatch entire intrinsic groups to another file.